README.md

```diff
@@ -1,12 +1,44 @@
 # realtime-metrics
+
 Evaluate recorded GTFS-RT data
 
-Install the package:
+## Example usage
+
+[GTFSDB](https://github.com/OpenTransitTools/gtfsdb) (to load the dataset into the db) is needed:
+
+
+```shell
+pip install git+https://github.com/1Maxnet1/gtfsdb.git@patch-1 setuptools # workaround until dependency issues are resolved upstream
+```
+
+Now one download a GTFS dataset and load it into the database:
+
 ```shell
+wget -O BUCHAREST-REGION.zip https://gtfs.tpbi.ro/regional/BUCHAREST-REGION.zip
+gtfsdb-load --database_url sqlite:///example.db BUCHAREST-REGION.zip
+```
+
+Now install `realtime-metrics` package:
+
+> [!NOTE]  
+> Currently GTFSdb works only with SQLAlchemy version 1.x, while realtime-metrics requires SQLAlchemy version 2.x
+> Therefore they cannot work in the same environment at the same time
+
+```shell
+pip uninstall sqlalchemy
 pip install -e .
 ```
 
-Run the script
+This does also install [gtfsrdb](https://github.com/public-transport/gtfsrdb). 
+
+Afterwards, use gtfsrdb to collect some realtime data:
+
+```shell
+gtfsrdb -t https://gtfs.tpbi.ro/api/gtfs-rt/tripUpdates -v https://gtfs.tpbi.ro/api/gtfs-rt/vehiclePositions -d sqlite:///example.db -c
+```
+
+Now you can run the script to calculate some metrics on it:
+
 ```shell
-python ./src/realtime_metrics/main.py -d sqlite:///test.db 
+python ./src/realtime_metrics/main.py -d sqlite:///example.db 
```

Comment:
Currently with no cli option, the script does not print anything for me. I would suggest to change the resulting statistics to a print statement instead of logger, to make sure they appear, even if -v is not set. Otherwise it is not quiet clear, what the script actually did.

Reply:
it now prints the MSE and ETA accuracies 👍🏼